adaptive-education-ai/src/ibm-watson/SaoTypeQuestionModel.ts

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+export class SaoTypeQuestionModel {
+    sentence: string;
+    question: string;
+    answer: string;
+    multipleChoice: any;
+
+    constructor(sentence: string, question: string, answer: string, multipleChoice: any) {
+        this.sentence = sentence;
+        this.question = question;
+        this.answer = answer;
+        this.multipleChoice = multipleChoice;
+    }   
+}
+
+export default SaoTypeQuestionModel;
```

Comment:
Not up to date on my import syntax, but is it valid to do this as `export default class SaoTypeQuestionModel` or something similar, rather than having a second line?